website/content/commands/license.mdx

```diff
@@ -52,11 +56,67 @@ Usage: consul license <subcommand> [options] [args]
   For more examples, ask for subcommand help or view the documentation.
 
 Subcommands:
-    get      Get the current license
-    put      Puts a new license in the datacenter
-    reset    Resets the license in the datacenter
+    get        Get the active license
+    inspect    inspect a signed license
```

Comment:
Maybe capitalize "Inspect" for consistency

Reply:
Yes it should. Thanks. Its updated now.